pkg/helm/chainlink/chainlink.go

```diff
@@ -120,6 +120,10 @@ func defaultProps() map[string]any {
 			},
 		},
 		"db": map[string]any{
+			"image": map[string]any{
+				"image":   "795953128386.dkr.ecr.us-west-2.amazonaws.com/postgres",
```

Comment:
can we mention the aws account directly?

Reply:
We already are higher up for the chainlink image so I am not adding anything new at the moment. Yes we need to get rid of these at some point.